fabtests/common/shared.c

```diff
@@ -3076,6 +3114,9 @@ void ft_parsecsopts(int op, char *optarg, struct ft_opts *opts)
 	case 'S':
 		if (!strncasecmp("all", optarg, 3)) {
 			opts->sizes_enabled = FT_ENABLE_ALL;
+		} else if (!strncasecmp("r:", optarg, 2)){
+			opts->sizes_enabled = FT_ENABLE_ALL;
```

Comment:
We might need to rename `FT_ENABLE_ALL` to something else.

Reply:
Renamed to FT_SIZES_ALL